bundles/org.openhab.binding.mybmw/src/test/java/org/openhab/binding/mybmw/internal/handler/auth/AuthTest.java

```diff
@@ -269,11 +280,11 @@ void testAuth() {
             chargingControlRequest.header("accept", "application/json");
             chargingControlRequest.header("x-user-agent", "android(v1.07_20200330);bmw;1.7.0(11152)");
             chargingControlRequest.header("accept-language", "de");
-            chargingControlRequest.header("Content-Type", CONTENT_TYPE_JSON_ENCODED);
+            chargingControlRequest.header("Content-Type", CONTENT_TYPE_JSON);
 
-            // String content = FileReader.readFileInString("src/test/resources/responses/charging-profile.json");
+            // String content = FileReader.readFileInString("responses/charging-profile.json");
```

Comment:
Can this commented line and the lines below be removed?

Reply:
done